.gitignore

```diff
@@ -34,27 +34,8 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# C extensions
-#*.so
-
-# Distribution / packaging
-.Python
-env/
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-*.egg-info/
-.installed.cfg
-*.egg
-.cache
+# Ignore compiled linux libs in the root folder, e.g. librust_g.so
```

Comment:
@ZeWaka Could you take a look through the .gitignore changes and make sure relevant? Relevant PR for comparison: https://github.com/tgstation/tgstation/pull/57326

Reply:
I know for sure the cache one is needed, style was lamenting about it for *minutes*

Reply:
They have `.cache` further down in their `.gitignore` under `# Unit test / coverage reports`, we don't have such a section. Running `bin/tgui --test` doesn't seem to generate any files that show in `git status`, so unsure if we need to re-add.

Reply:
eh whatever really